test/system/src/test/java/io/pravega/test/system/DynamicRestApiTest.java

```diff
@@ -140,4 +155,42 @@ public void listScopes() {
         responseAsString = response.readEntity(String.class);
         assertTrue(responseAsString.contains(String.format("\"streamName\":\"%s\"", stream1)));
     }
+
+    private static SSLContext buildSSLContext() {
+        try {
+            final SSLContext context = SSLContext.getInstance("TLSv1.2");
+            final TrustManager[] trustManagerArray = {new NullX509TrustManager()};
+            context.init(null, trustManagerArray, null);
+            return context;
+        } catch (Exception ex) {
+            ex.printStackTrace();
+        }
+        return null;
+    }
+
+    /**
+     * Host name verifier that does not perform nay checks.
+     */
+    private static class NullHostnameVerifier implements HostnameVerifier {
+        public boolean verify(String hostname, SSLSession session) {
+            return true;
+        }
+    }
+
+    /**
+     * Trust manager that does not perform nay checks.
+     */
+    private static class NullX509TrustManager implements X509TrustManager {
```

Comment:
maybe we need a TODO here that actually checks the server is trusted with the truststore configured correctly.

Reply:
kept the TODO comments.